weaveworks-nginx/profile.yaml

```diff
@@ -24,3 +24,17 @@ spec:
         defaultValues: |
           service:
             type: ClusterIP
+    - name: dependon-chart
+      chart:
+        url: https://charts.bitnami.com/bitnami
+        name: redis
+        version: "14.6.1"
+        defaultValues: |
+          master:
+            persistence:
+              enabled: false
+          replica:
+            replicaCount: 0
```

Comment:
lol

Reply:
Yeah this is needed for redis to properly start without any problems and extras. :D Also, we do need SOME diversity in the stuff we install. :DD